src/data_mover_threads.c

```diff
@@ -215,8 +231,26 @@ static struct vdm data_mover_threads_vdm = {
  */
 struct data_mover_threads *
 data_mover_threads_new(size_t nthreads, size_t ringbuf_size,
+	struct data_mover_threads_op_fns *op_fns, size_t ops_size,
 	enum future_notifier_type desired_notifier)
 {
+	if (sizeof(struct data_mover_threads_op_fns) < ops_size) {
+		/*
+		 * The caller was compiled using a newer version of
```

Comment:
Do we even need it if we don't have any miniasync version released?

Reply:
It's better to add this now and forget about it than to omit it and forget about it when we release miniasync, potentially breaking ABI compatibility.